_config/config.yml

```
@@ -0,0 +1,11 @@
+---
+Name: tinymce-config
+---
+SilverStripe\Forms\HTMLEditor\HTMLEditorConfig:
+  default_config_definitions:
+    cms:
+      configClass: 'SilverStripe\TinyMCE\TinyMCEConfig'
```

```suggestion
      configClass: SilverStripe\TinyMCE\TinyMCEConfig
```
Don't need quotes for an FQCN

---

Discussion about that can be in https://github.com/silverstripe/silverstripe-framework/pull/11617#discussion_r1988461686